docs/getting-started/install-endpoint.asciidoc

```diff
@@ -107,6 +107,19 @@ After you have installed the agent, malware prevention is automatically enabled
 [role="screenshot"]
 image::images/install-endpoint/malware-protection.png[]
 
+## Register as antivirus (Windows Only)
+
+If you download the Elastic Agent on Windows 10 or above, you can Elastic Security as your antivirus software. To enable Elastic Security as antivirus: 
```

Comment:
```suggestion
If you download the Elastic Agent on Windows 10 or above, you can configure Elastic Security as your antivirus software by doing the following:
```

Reply:
Consider changing `download` -> `downloaded` because they likely will have done so by the time they get to this step.